### ISSUE TITLE
[Leveler] Exception with [p]top command

### ISSUE THREAD

**Issue by PhasecoreX** [fixator10/Fixator10-Cogs#94]
**Describe the bug**

Exception thrown with the `[p]top` command.

**To Reproduce**

Seems like you just have to run the `[p]top` command when you haven't sent any other messages in the server?

**Expected behavior**

No exception thrown, show the expected output of `[p]top`.

**Bot's logs**

```python
Traceback (most recent call last):
  File "/data/venv/lib/python3.8/site-packages/discord/ext/commands/core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "/data/cogs/CogManager/cogs/leveler/leveler.py", line 390, in top
    await self._find_server_exp(user, server),
  File "/data/cogs/CogManager/cogs/leveler/leveler.py", line 3583, in _find_server_exp
    for i in range(userinfo["servers"][str(server.id)]["level"]):
TypeError: 'NoneType' object is not subscriptable

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/data/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 939, in invoke
    await ctx.command.invoke(ctx)
  File "/data/venv/lib/python3.8/site-packages/discord/ext/commands/core.py", line 863, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "/data/venv/lib/python3.8/site-packages/discord/ext/commands/core.py", line 94, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: 'NoneType' object is not subscriptable
```

**Red version**

3.4.9

**Additional context**

Happened in my server with a member who just joined the server. I can run the `[p]top` command just fine, but when they ran it the exception above happened instead. I did a quick glance at the code and it's probably because `userinfo` on line 3580 is None, since they never sent any messages beforehand.

**Comment by fixator10** [comment]
Should be fixed with 399f213bd0fe35d2c05048ec1805c6baa8be1b6b, can you check pls?

**Comment by PhasecoreX** [comment]
Sorry busy day, yes this seems to fix it, thank you!
